### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by amerkoleci** [amerkoleci/Vortice.Windows#293] [Q&A] [answered]
Anyone knows a way to delete old github actions?

I would like to keep last 30 days of CI workflow builds?

c.c: @Axiverse @paulbartrum @miyu @h1cks 

Thanks

**Comment by h1cks** [comment]
Looking through the documentation I could find, this has been asked for a few times.  But not supported.  But I couldnt find anything else.  Not a definitive answer. Sorry.

**Comment by miyu** [comment, marked as answer]
Is this helpful? https://github.com/Mattraks/delete-workflow-runs

**Reply by amerkoleci** [reply]
Nice, works well.

Thanks!
